lumicks/pylake/tests/test_imaging_confocal/test_kymo.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+import pytest
+from lumicks.pylake.channel import Continuous, Slice
+
+from ..data.mock_confocal import (
+    axes_dict_list,
+    confocal_mock_file_h5,
+    generate_kymo_with_ref,
+    generate_scan_json,
+)
+
+
+# CAVE: If you want to test a cached property, after having modified parameters that change the
+# value of the property, ensure to clear the `_cache` attribute before and after testing. To achieve
+# both, you can monkeypatch the `_cache` attribute with an empty dict.
+@pytest.fixture(scope="module")
+def test_kymos():
+    return {"standard": generate_kymo_with_ref("standard", np.random.poisson(10, (5, 4, 3)))}
+
+
+@pytest.fixture
+def add_force_channel(monkeypatch):
+    def do_add_force_channel(kymo, ref, head=30, tail=10, head_lines=2, channel="force1x"):
```

Comment:
Minor nitpick, but I wouldn't be able to guess from `head` and `tail` what they'd do. `early_force`, `late_force`? I also wonder if `attribute` is a better descriptor than `channel` here, since you're not really adding a channel, but specifically that attribute.